test/ProxyEpochRecordFilterTest.cpp

```diff
@@ -64,6 +68,8 @@ void ProxyEpochRecordFilterTest::SetUp()
     m_out_events = {
         geopm::EVENT_EPOCH_COUNT,
     };
+    m_tutorial_2_prof_trace_path = GEOPM_SOURCE_DIR;
+    m_tutorial_2_prof_trace_path += "/test/ProxyEpochRecordFilterTest.tutorial_2_profile_trace";
```

Comment:
In the future, we may want to have other tests that need trace file data as input.  When that happens, it would be a nice refactor to have one class member like ```std::string m_prof_trace_path``` and then have the specific tests populate the rest of the path to the data then parse.